### PR TITLE
Emulate APIv2 call for playlist_items

### PR DESCRIPTION
**Pull request by VioletRed**
Start playlist from video_id passed as argument.

Allow "/play/?playlist_id=XXXX&video_id=YYYYY" as URI, to match old plugin functionality.

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.youtube" name="YouTube" version="5.0.1" provider-name="bromix">
+<addon id="plugin.video.youtube" name="YouTube" version="5.0.2" provider-name="bromix">
```
Comment (bromix) [inline addon.xml]: Please do not change version numbers ;)

**Comment by bromix** [comment]
I have to ask, what has this to do with APIv2 (by Google)? Who is calling this url to play a playlist from a particular video item?

**Comment by VioletRed** [comment]
APIv2 allowed to get playlist items from a specific item, in URI like "https://www.youtube.com/watch?v=qmlYe2KS0-Y&index=6&list=PLZRRxQcaEjA5fgfW3a3Q0rzm6NgbmICtg", and old youtube plugin would process a URI such "plugin://plugin.video.youtube/?action=play_all&playlist=PLZRRxQcaEjA5fgfW3a3Q0rzm6NgbmICtg&videoid=qmlYe2KS0-Y", meaning "reproduce playlist PLZRRxQcaEjA5fgfW3a3Q0rzm6NgbmICtg starting at video qmlYe2KS0-Y."

 

APIv3 won't return a list in such situation, but only the given video if it exist in the playlist.

 

About who was using it, I was using it in my Greasemonkey XBMC integration script.

 

Sent: Thursday, January 01, 2015 at 5:43 PM

**Comment by bromix** [comment]
Please consider https://github.com/bromix/plugin.video.youtube/commit/3f78092d27d8b0a63764c65c2a4ea0ba5d6249fa my solution for this (#129). Via my repo the 5.0.2 can already be tested. Thanks for your information....I didn't know about this. I also added the shared playlists support via Yatse.

**Pull request closed without merging**